### DISCUSSION TITLE
提问，这里是罗浮杂俎吗？

### DISCUSSION THREAD

**Discussion by tyhpoi** [share121/inter-knot#1707] [其他]
![微信图片_20240812170824](https://github.com/user-attachments/assets/0b0997e3-29ff-43a8-a107-863768b2876d)

我记得这是素裳给的网址啊

**Comment by Elysia-Eden-11** [comment]
不是

**Comment by Merlin-cyq** [comment]
不是，你进错网站了

**Comment by Aventurinefragments** [comment]
大兄弟，你上错车啦

**Comment by XokoukiguraX** [comment]
罗浮杂俎怎么走？

**Comment by XokoukiguraX** [comment]
罗浮杂俎怎么走？

**Comment by NorthernLightCN** [comment]
你这个素裳。。。是不是经营着狡兔屋那个？

**Comment by zhangyj0523** [comment]
好主意，以后做个ui切换如何

**Comment by 1451374815** [comment]
错了，正确的网址是https://www.bilibili.com/video/BV1GJ411x7h7/?spm_id_from=333.337.search-card.all.click&vd_source=071a2edf4de47e8535e65e37ec58abad

**Reply by TW0618** [reply]
可恶，被骗了
![70B74EBD](https://github.com/user-attachments/assets/c5d3fe10-617d-47a2-9b1e-418320986d15)

**Reply by ikungei** [reply]
经典骗局😡

**Comment by lamtt26** [comment]
不知道啊，有谁知道吗

**Comment by sfhjhsdfg** [comment]
你这提醒我了，可以仿制一个罗浮杂俎，~~记小本本~~未来学一学

**Comment by RXLX-CHINA** [comment]
诶呀，你大爷啊，给我干哪来了，这还是仙舟吗？

**Comment by gvfjh** [comment]
错了，罗浮杂俎的网址是[罗浮杂俎首页](https://www.bilibili.com/video/BV1GJ411x7h7)

**Reply by wum2008** [reply]
可恶，又被骗了

**Comment by 206288602** [comment]
那几个网址，还真是欢愉

**Comment by qwer729** [comment]
奶奶滴，被骗了

**Comment by TW0618** [comment]
又被骗了。。。

**Comment by YWphosphenes02** [comment]
可恶 到了绳网也要被骗

**Comment by hanyaser** [comment]
又被骗了_(:з」∠)_

**Comment by wum2008** [comment]
两次啊，两次
![Image_1723574179150](https://github.com/user-attachments/assets/46b2b11c-26d3-49e9-851a-4a3f9621d4b8)

**Comment by CocoonedSoul** [comment]
不是，我都看见被骗的字了，为啥还要点进去？二重奏听的我旋转跳跃。
